Assets/MixedRealityToolkit.SDK/Features/Utilities/Solvers/HandConstraintPalmUp.cs

```diff
@@ -15,17 +16,18 @@ public class HandConstraintPalmUp : HandConstraint
     {
         [Header("Palm Up")]
         [SerializeField]
+        [FormerlySerializedAs("facingThreshold")]
         [Tooltip("The angle (in degrees) of the cone between the palm's up and camera's forward have to match. Only supported by IMixedRealityHand controllers.")]
         [Range(0.0f, 90.0f)]
-        private float facingThreshold = 80.0f;
+        private float facingCameraTrackingThreshold = 80.0f;
 
         /// <summary>
         /// The angle (in degrees) of the cone between the palm's up and camera's forward have to match. Only supported by <see cref="Microsoft.MixedReality.Toolkit.Input.IMixedRealityHand"/> controllers.
         /// </summary>
-        public float FacingThreshold
+        public float FacingCameraTrackingThreshold
```

Comment:
I believe this is actually a [breaking change](https://microsoft.github.io/MixedRealityToolkit-Unity/Documentation/Contributing/BreakingChanges.html) since it is a change to a public property in a sdk asset. The new naming does seem more descriptive which is a positive. A drawback is that people may get compile errors if they update. This might be a case [where the breaking change adds value but can be written in a way that isn't breaking](https://microsoft.github.io/MixedRealityToolkit-Unity/Documentation/Contributing/BreakingChanges.html#the-breaking-change-adds-value-but-could-be-written-in-a-way-that-isnt-breaking).

What are your thoughts on the value vs drawbacks of this rename?

Reply:
My mistake! I changed the private value and add the formerly serialized attribute to persist user's previous values in existing prefabs.

To be honest, I don't many folks would be changing this value at runtime via a script. Nothing in the MRTK does so, so it would be entirely in user app level code. If you want me to revert it, I'll do so, but I doubt many folks will notice.

Reply:
I think the serialized field can be modified but for the public property make it twice

```
public float FacingCameraTrackingThreshold
{
}
[System.Obsolete("Use FacingCameraTrackingThreshold property instead")
public float FacingThreshold
{
}